src/TLuaInterpreter.cpp

```diff
@@ -2462,9 +2472,15 @@ int TLuaInterpreter::openUserWindow( lua_State *L )
     {
         luaSendText = lua_tostring( L, 1 );
     }
+
+    bool loadLayout = true;
+    if( ! lua_isnoneornil(L, 2) && lua_isboolean(L, 2) ) {
```

Comment:
I forget where I found that in the source now. There are definitely other lua API members that use it, and it is handy for optional arguments. :)